labelbox/schema/dataset.py

```diff
@@ -260,11 +260,10 @@ def upload_if_necessary(item):
             row_data = item['row_data']
             if os.path.exists(row_data):
                 item_url = self.client.upload_file(item['row_data'])
```

Comment:
**suggestion** can we change this `item_url = self.client.upload_file(item['row_data'])` to this?

`item_url = self.client.upload_file(row_data)`